Norma-Portuguesa-405-citação-numérica

```diff
@@ -0,0 +1,522 @@
+        <text variable="page" prefix=". p. "/>
+      </else-if>
+      <else-if type="book" match="any">
+        <text variable="page" prefix=". " suffix=" p."/>
+      </else-if>
+      <else-if type="thesis" match="any">
+        <text variable="page" prefix=". " suffix=" f."/>
+      </else-if>
+      <else-if type="report" match="any">
+        <text variable="number" prefix=" (Relatório n." suffix=")."/>
+      </else-if>
+      <else-if type="article-newspaper" match="any">
+        <date variable="issued" prefix=" (" suffix="). " delimiter=" ">
+          <date-part name="day"/>
+          <date-part name="month" form="short" suffix="." text-case="lowercase"/>
+          <date-part name="year"/>
+        </date>
+        <text variable="volume" suffix=":"/>
+        <text variable="issue"/>
+        <text variable="page" prefix=" "/>
+      </else-if>
+      <else-if type="webpage" match="any">
+        <date variable="issued" prefix=", atual. ">
+          <date-part name="day" suffix=" "/>
+          <date-part name="month" form="short" suffix=". " text-case="lowercase"/>
+          <date-part name="year" suffix=". "/>
+        </date>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="ISBN">
+    <group>
+      <text variable="ISBN" prefix=". ISBN "/>
+    </group>
+  </macro>
+  <macro name="ISSN">
+    <group>
+      <text variable="ISSN" prefix="ISSN "/>
+    </group>
+  </macro>
+  <macro name="archive">
+    <group>
+      <text variable="archive" prefix=" "/>
+    </group>
+  </macro>
+  <citation et-al-min="4" et-al-use-first="1" collapse="year">
+    <sort>
+      <key variable="citation-number"/>
+    </sort>
+    <layout delimiter="; ">
+      <text variable="citation-number" vertical-align="sup"/>
+    </layout>
+  </citation>
+  <bibliography hanging-indent="false" et-al-min="4" et-al-use-first="1" entry-spacing="0">
+    <layout>
+      <text variable="citation-number" suffix=". "/>
+      <choose>
+        <if type="bill legislation" match="any">
+          <group>
+            <text variable="title" suffix=". "/>
+            <text variable="collection-title" suffix=". "/>
+            <text macro="container-title"/>
+            <text macro="container-contributors" suffix=". "/>
+            <text macro="medium"/>
+            <text macro="locators" suffix=" "/>
+            <text macro="access"/>
+          </group>
+        </if>
+        <else-if type="map">
+          <group>
+            <text macro="author" suffix=" - "/>
+            <text macro="title"/>
+            <text macro="medium"/>
+            <text macro="edition" prefix=". "/>
+            <text macro="locators"/>
+            <text macro="access" prefix=" "/>
+            <text variable="note" suffix=". "/>
+          </group>
+        </else-if>
+        <else-if type="book">
+          <group>
+            <text macro="author" suffix=" - "/>
+            <text macro="title"/>
+            <text variable="collection-title" suffix=". "/>
```

Comment:
avoid mixing suffix and prefix. Here, an item with collection-title and edition would have ". . " in the reference, for example. Same applies for all the style. The ideal solution would be a group with a ". " delimiter, but absent that, use prefixes or suffixes consistently within any item type.